configuration/concepts/commons.csv

```diff
@@ -139,4 +139,9 @@ c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Visit number,Numéro de la visite,Visit n
 c4b48a7a-7c13-4e7d-89ad-1df68c5c6801,,,First Visit,Première visite,First Visit,Première visite,,,Misc,Coded,,bf885a9d-9f98-4990-8c20-ac4ace816bfe;2120763c-6c49-4133-b532-1e931ab4858a,,,,,,,,,
 f5e7648d-9dfb-4506-9602-000ab940a197,,,Quantity Dispensed,Quantité distribuée,Quantity Dispensed,Quantité distribuée,,,Misc,Numeric,,,,TRUE,,,,,,,
 9d20f065-9568-4769-80b5-f596907a8805,,,Details,Précisions,Details,Précisions,,,Misc,Text,,,,,,,,,,,
-2da93612-9727-4a29-ae6d-518fc62265ae,,,All Observation Templates,All Observation Templates,,,,,ConvSet,N/A,,,,,,,,,,,
\ No newline at end of file
+2da93612-9727-4a29-ae6d-518fc62265ae,,,All Observation Templates,All Observation Templates,,,,,ConvSet,N/A,,,,,,,,,,,
+7a5b55b6-4617-42b9-808c-82116a259a6e,,,Minibus,Tap Tap,Minibus,Tap Tap,,,Misc,N/A,,,,,,,,,,,
```

Comment:
Was this "Tap tap" in english or in french originally?
Anyway it should probably stay "Tap tap" in all languages.

Reply:
Done